configuration/pih/htmlforms/hiv/hiv-dispensing.xml

```diff
@@ -615,6 +617,8 @@
                                 TDF/3TC 300/300mg,
 
                                 ABC/3TC 120/60mg,
+                                DTG 50mg,
+                                DTG 5mg,
```

Comment:
@louidorjp Looks good, but I would suggest changing the order so it's 50, 10, 5mg (instead of 50, 5, 10) and same order as the ARV #2